torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -1361,12 +1361,16 @@ std::vector<Val*> Index::getGlobalProducerStridedIndices(
 
     TORCH_INTERNAL_ASSERT(
         root_ind != nullptr,
-        "Couldn't find root mapping for TV",
-        producer_tv->name(),
+        "Couldn't find root mapping for ",
```

Comment:
The changes of this assert and the other similar ones in this file are not directly related to the fix, but they were missing some useful information and also showing wrong information.